app/src/test/java/com/simply/schedule/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.simply.schedule
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
I don't think it's necessary to pay too much attention to the testing now, but it's good you did a mock